No ARM support?

We're attempting to publish and sign binaries for Windows x64, x86 and ARM64. 
We're running into an issue where we can sign applications on the `windows-2025` runner, but not on the `windows-11-arm` runner.

This is the log output I'm getting on the ARM64 runner:

```
Checking for required dependencies.
	Build tools package installed: False
	Trusted signing package installed: False
	Sign CLI package installed: False
	Installing required dependencies.
		Found existing package source: https://www.nuget.org/api/v2/
		Installing package: Microsoft.Windows.SDK.BuildTools 10.0.22621.3233
		Installing package: Microsoft.Trusted.Signing.Client 1.0.53
		Installing package: sign 0.9.1-beta.24469.1

Creating metadata file: C:\Users\runneradmin\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.53\bin\x64\metadata.json

Getting the list of files to be signed.
	Getting files from list.
	Listed files: 1
		<path to our .exe>

	No files folder was provided.
	Filtered files: 0

	No catalog file was provided.
	Catalog files: 0

Formatting the list of files to be signed.

Batching the list of files to be signed by SignTool.
	Batched SignTool files: 1

Batching the list of files to be signed by SignCli.
	No SignCli files were found to sign.

Signing SignTool file batch 1 of 1.
	Executing signtool.exe: C:\Users\runneradmin\AppData\Local\TrustedSigning\Microsoft.Windows.SDK.BuildTools\Microsoft.Windows.SDK.BuildTools.10.0.22621.3233\bin\10.0.22621.0\x64\signtool.exe sign /v /debug /fd SHA256 /tr http://timestamp.acs.microsoft.com/ /td SHA256 /dlib "C:\Users\runneradmin\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.53\bin\x64\Azure.CodeSigning.Dlib.dll" /dmdf "C:\Users\runneradmin\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.53\bin\x64\metadata.json" "<path to our exe>"
Exception: SignTool failed with exit code 3
Error: Process completed with exit code 1.
````

I suspect the issue is that arm is not properly detected and the x64 version of the binary (`Microsoft.Trusted.Signing.Client.1.0.53\bin\x64\Azure.CodeSigning.Dlib.dll`) is being invoked?

This is how we invoke the action:

```yaml
- name: 🖋️ Sign Windows binary
  uses: azure/trusted-signing-action@v0
  with:
    endpoint: ${{ vars.AZURE_TRUSTED_SIGNING_ACCOUNT_ENDPOINT }}
    trusted-signing-account-name: ${{ secrets.AZURE_CODE_SIGNING_NAME }}
    certificate-profile-name: ${{ secrets.WINDOWS_APP_CERT_PROFILE_NAME }}
    files: ${{ github.workspace }}\publish\nitro.exe
    file-digest: SHA256
    timestamp-rfc3161: http://timestamp.acs.microsoft.com
    timestamp-digest: SHA256
    exclude-environment-credential: true
    exclude-workload-identity-credential: true
    exclude-managed-identity-credential: true
    exclude-shared-token-cache-credential: true
    exclude-visual-studio-credential: true
    exclude-visual-studio-code-credential: true
    exclude-azure-cli-credential: false
    exclude-azure-powershell-credential: true
    exclude-azure-developer-cli-credential: true
    exclude-interactive-browser-credential: true
```

--- a/README.md
+++ b/README.md
@@ -11,6 +11,8 @@ This Action can only be executed on Windows runners. It is supported by the foll
 - [windows-2022](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md)
 - [windows-2019](https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md)
 
+> We currently do not support Windows Arm runners.
+
 It is also possible to use self-hosted runners with the following requirements:
 - Windows 7+
 - PowerShell 5.1+

